smdebug/profiler/system_profiler_file_parser.py

```diff
@@ -51,16 +52,34 @@ def read_events_from_file(self, eventfile):
         try:
             with open(eventfile) as json_data:
                 event_line = json_data.readline()
+                lineNum = 0
                 while event_line:
+                    lineNum += 1
                     json_event = json.loads(event_line)
                     event_line = json_data.readline()
                     self.read_event_from_dict(json_event)
         except Exception as e:
+            with open(eventfile) as j_data:
+                event_line = j_data.readline()
+                lineNumTotal = 0
+                while event_line:
+                    event_line = j_data.readline()
+                    lineNumTotal += 1
+            errStr = 'line No. ' + str(lineNum) + ' out of total ' + str(lineNumTotal) + ' lines, '
+            cur_dir = os.path.dirname(eventfile)
+            cur_dir_base = os.path.basename(cur_dir)
+            eventfile_base = os.path.basename(eventfile)
+            parent_dir = os.path.dirname(cur_dir)
+            time_stamp_files = [int(x.split('.')[0]) for x in os.listdir(cur_dir)]
+            time_stamp_dirs = [int(x) for x in os.listdir(parent_dir)]
+            file_pos = time_stamp_files.index(int(eventfile_base.split('.')[0]))
+            dir_pos = time_stamp_dirs.index(int(cur_dir_base))
+            errStr += 'file No.' + str(file_pos+1) + ' in file No.1 to No.' + str(len(time_stamp_files)) + ', folder No.' + str(dir_pos+1) + " in folder No.1 to No." + str(len(time_stamp_dirs)) + "\n"
             self.logger.error(
-                f"Can't open profiler system metric file {eventfile}: Exception {str(e)}"
+                f"{errStr} Can't open profiler system metric file {eventfile}: Exception {str(e)}"
```

Comment:
Don't print this at the start of the message, print it somewhere in the middle or the end, so that customers can quickly categorize the error